package.json

```diff
@@ -62,10 +64,9 @@
     "react-dom": "18.2.0",
     "react-dropzone": "14.2.3",
     "react-portal": "4.2.2",
-    "react-redux": "8.0.5",
+    "react-redux": "9.0.4",
```

Comment:
Regarding the selector error, this is most likely the culprit. Is upgrade to version 9 required for vite upgrade to work?

Reply:
Huh, that worked! Thanks mate :)

Reply:
Great!